sqlalchemy_filters/filters.py

```diff
@@ -91,9 +137,11 @@ def apply_filters(query, filters):
     if not models:
         raise BadQuery('The query does not contain any models.')
 
-    sqlalchemy_filters = [
-        Filter(filter_, models).format_for_sqlalchemy() for filter_ in filters
-    ]
+    if not isinstance(filters, list):
+        raise BadFilterFormat('`filters` must be a list')
```

Comment:
I added this validation, as otherwise (due to the nature of `build_sqlalchemy_filters`) it would be possible to just do `filters={'field': 'id', 'op': ...}` or `filters={'or': [{...}, {...}]}` which might confuse the API?